lib/ansible/modules/apt.py

```diff
@@ -563,7 +563,7 @@ def expand_dpkg_options(dpkg_options_compressed):
     return dpkg_options.strip()
 
 
-def expand_pkgspec_from_fnmatches(m, pkgspec, cache):
+def expand_pkgspec_from_fnmatches(m, pkgspec, cache, to_rmv=False):
```

Comment:
```suggestion
def expand_pkgspec_from_fnmatches(m, pkgspec, cache, remove=False):
```

Reply:
@s-hertel Thank you for the review! I've changed the name of internal variable and rebased my branch.